server/views/pages/article.njk

```diff
@@ -76,12 +76,14 @@
 {% block transporters %}
   {% for series in article.series %}
     {% if series.commissionedLength %}
-      <div class="container digital-story-transporter">
-        <div class="grid grid--no-gutters">
-          <div class="{{ {s: 4, m: 6, shiftM: 1, l: 7, xl: 6, shiftXl: 1} | gridClasses }}">
-            <div class="body-part body-part--full-width">
-              {% set asyncContentEndpoint = '/series-transporter/' + series.url + '?current=' + article.url %}
-              {% component 'async', { endpoint: asyncContentEndpoint, component: 'series-transporter' } %}
+      <div class="row {{ {s:4, m:4, l:4} | spacingClasses({padding: ['bottom']}) }}">
+        <div class="container digital-story-transporter">
+          <div class="grid grid--no-gutters">
+            <div class="{{ {s: 4, m: 6, shiftM: 1, l: 7, xl: 6, shiftXl: 1} | gridClasses }}">
+              <div class="body-part body-part--full-width">
+                {% set asyncContentEndpoint = '/series-transporter/' + series.url + '?current=' + article.url %}
+                {% component 'async', { endpoint: asyncContentEndpoint, component: 'series-transporter', modifiers: '{"in-article": true, "with-wobbly-line": true}' } %}
```

Comment:
Could we not do something in the logic of the component to add the modifiers?
This seems like a very specific piece of functionality wedged onto a generic component.